guides/source/developers/extensions/writing-extensions.html.md

```diff
@@ -0,0 +1,67 @@
+# Writing a New Solidus Extension
+
+Sometimes building a site powered by Solidus will require customization.
+Solidus extensions are Rails engines that allow you to extend or change the
+functionality of the Solidus platform.
+
+## Find existing extensions
+
+Before you write a new Solidus extension you should look for an existing
+extension that meets your needs. Solidus maintains an official list of
+[supported extensions](http://extensions.solidus.io), but you can also use
+[soliton](http://soliton.nebulab.it) to search for extensions.
+
+## Developing a Solidus extension locally
+
+### Install the solidus_cmd gem
+To develop your own Solidus extension, you'll need to install the
```

Comment:
One does not need to install this gem. It is a nice little wrapper around `gem new` or [`bundle gem`](https://bundler.io/v1.16/guides/creating_gem.html).

Reply:
Hey Thomas, great feedback! I addressed most of your thoughts in my last commit, but I'm not sure I understood this comment. I sent you a Slack message about it, but you're also welcome to explain here. 

I'm not sure how someone would use the `solidus_cmd` gem without installing it. Could you explain how it should be used?